### PR TITLE
kind: add page

### PR DESCRIPTION
**Pull request by thamaraiselvam**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/kind.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# kind
+
+> Kind is a tool for running local Kubernetes clusters using Docker container "nodes".
+> Kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
```
Comment (sbrl) [inline pages/common/kind.md:4]: Maybe we could have this more concise?
```suggestion
> Tool for running local Kubernetes clusters using Docker container "nodes".
> Designed for testing Kubernetes itself, but may be used for local development or continuous integration.
```

File: pages/common/kind.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`kind create cluster --name {{cluster-name}}`
+
+- Delete one or more clusters:
+
+`kind delete clusters {{cluster-name}}`
+
+- Get one of [clusters, nodes, kubeconfig]:
+
+`kind get clusters`
```
Comment (sbrl) [inline pages/common/kind.md:17]: How does one determine which one to fetch? If it's by the keyword there, what about this?
```suggestion
- Get details about clusters, nodes, or the kubeconfig:

`kind get {{clusters|nodes|kubeconfig}}`
```

File: pages/common/kind.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- Export one of [kubeconfig, logs]:
+
+`kind export logs`
```
Comment (sbrl) [inline pages/common/kind.md:21]: Same here.
```suggestion
- Export the kubeconfig or the logs:

`kind export {{kubeconfig|logs}}`
```

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page, @thamaraiselvam! I've left some comments below for you to review.

**Comment by thamaraiselvam** [comment]
@sbrl updated changes

**Review by sbrl — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**